frontend/js/src/settings/missing-data/MultiTrackMBIDMappingModal.tsx

```diff
@@ -0,0 +1,625 @@
+                    data-tip
+                    data-for="musicbrainz-helptext"
+                    size="sm"
+                  />
+                </p>
+                <div className="card listen-card">
+                  <SearchAlbumOrMBID
+                    key={searchTerm}
+                    onSelectAlbum={setSelectedAlbumMBID}
+                    defaultValue={searchTerm ?? ""}
+                  />
+                </div>
+                <div className="form-check mt-15">
+                  <input
+                    className="form-check-input"
+                    id="includeArtistSearch"
+                    type="checkbox"
+                    checked={includeArtistNameSearch}
+                    onChange={(e) =>
+                      setIncludeArtistNameSearch(e.target.checked)
+                    }
+                  />
+                  &nbsp;
+                  <label
+                    htmlFor="includeArtistSearch"
+                    style={{ fontWeight: "initial" }}
+                  >
+                    Include artist name when searching{" "}
+                    <FontAwesomeIcon
+                      icon={faQuestionCircle}
+                      data-tip
+                      data-for="includeArtistNameHelp"
+                      size="sm"
+                    />{" "}
+                  </label>
+                </div>
+                <Tooltip id="includeArtistNameHelp" type="info" multiline>
+                  Depending on the data we have available, including the artist
+                  name can result in worse matching. Tick this checkbox if you
+                  have a poor matching rate.
+                </Tooltip>
+                <div className="form-check">
+                  <input
+                    className="form-check-input"
+                    id="escapeSpecialCharacters"
+                    type="checkbox"
+                    checked={escapeSpecialCharacters}
+                    onChange={(e) =>
+                      setEscapeSpecialCharacters(e.target.checked)
+                    }
+                  />
+                  &nbsp;
+                  <label
+                    htmlFor="escapeSpecialCharacters"
+                    style={{ fontWeight: "initial" }}
+                  >
+                    Preserve special characters{" "}
+                    <FontAwesomeIcon
+                      icon={faQuestionCircle}
+                      data-tip
+                      data-for="escapeSpecialCharactersHepl"
+                      size="sm"
+                    />{" "}
+                  </label>
+                </div>
+                <Tooltip id="escapeSpecialCharactersHepl" type="info">
+                  Escape special characters such as{" "}
+                  <span className="code-block">
+                    ( ) [ ] ! * ~ ^ &quot; ~ ? \ / || &&
+                  </span>
+                  to preserve them in the search term.
```

Comment:
Adding a space here. 
<img width="675" alt="image" src="https://github.com/user-attachments/assets/f598692d-b18f-41db-b62e-3d50aa8c4768">


```suggestion
                  </span>{" "}
```